tests/test_commit_message.py

```diff
@@ -104,6 +110,34 @@ def test_group_reviewers_replaced(reviewer_text):
     assert commit_message == (FIRST_LINE, COMMIT_MESSAGE)
 
 
+def test_group_reviewers_replaced_with_period_at_end():
+    """Test unexpected period after reviewer name.
+    """
+    # NOTE: the parser stops parsing after the period at the end of a reviewer
+    # name, therefore any other reviewers past the first period will not be
+    # parsed correctly, and the output will be mangled. This should be fixed
+    # and the test should be updated.
+
+    reviewers = ["reviewer_one", "reviewer.two"]
+    commit_message = format_commit_message(
+        "A title. r=a.,b", 1, reviewers, "A summary.", "http://phabricator.test/D123"
+    )
+
+    # This is the current behaviour
+    assert commit_message == (
+        "Bug 1 - A title. r=reviewer_one,reviewer.two.,b",
+        "Bug 1 - A title. r=reviewer_one,reviewer.two.,b\n\n"
+        "A summary.\n\nDifferential Revision: http://phabricator.test/D123",
+    )
+
+    # This is the desired future behaviour
+    assert commit_message != (
```

Comment:
For things like this you could always write the test as if it would pass but put an `@pytest.mark.xfail(strict=True)` so that we update the test when we fix things. See https://docs.pytest.org/en/latest/skipping.html#xfail

Reply:
I considered using that but wanted to include the passing case as well, but now that I think about it, I can still include the passing case for reference anyway! Changed.